node/channel.js

```diff
@@ -370,7 +370,8 @@ TChannel.prototype.request = function channelRequest(options) {
     }
 
     var req = null;
-    if (opts.host || // retries are only between hosts
```

Comment:
Since self.peers.request doesn't have resend for call req/res. 

@jcorbin, should I add a blocking and retry logic in v2/out_request.js?